discovery_engine_core/ai/xayn-ai/src/lib.rs

```diff
@@ -0,0 +1,41 @@
+#![forbid(unsafe_op_in_unsafe_fn)]
+
+mod coi;
+mod data;
+mod embedding;
+mod error;
+pub mod ranker;
+mod utils;
+
+pub use crate::{
+    coi::CoiId,
+    data::document::{
+        DayOfWeek,
+        Document,
+        DocumentHistory,
+        DocumentId,
+        QueryId,
+        Relevance,
+        SessionId,
+        UserAction,
+        UserFeedback,
+    },
+    error::Error,
+    embedding::utils::COSINE_SIMILARITY_RANGE,
```

Comment:
This is needed because otherwise it complains that we have reference to private types in the public documentations here: ai/xayn-ai/src/coi/config.rs